cloud-azure/src/test/java/com/sequenceiq/cloudbreak/cloud/azure/validator/AzureTagValidatorTest.java

```diff
@@ -151,5 +154,11 @@ PlatformParameters parameters() {
             when(mock.tagValidator()).thenReturn(azureTagValidator);
             return mock;
         }
+
+        @Bean
+        @Qualifier("AzureImageFilter")
+        ImageFilter azureImageFilter() {
+            return new CommonImageFilter();
```

Comment:
there is a new bean here but there is no assertion in the tests

Reply:
This bean is only needed because otherwise I'd need to import all the transitive dependencies of `AzureImageFilter` in this `@Configuration` which is an excessive list that I could not manage to include, because the likes of eg. `EntitlementService`

Image filtering is not tested at all here.

Reply:
discussed on a zoom call.